css/components/section-3.css

```diff
@@ -1,65 +1,65 @@
 
-#mw-section-3 {
+#section-3 {
```

Comment:
doar ca idee , m-am referit doar la id-uri sa scoti mw- din fatza nu la toate clasele.
De ce ?
In librarii precum bootstrap, nu o sa gasesti id-uri niciodata , doar clase.